src/app/components/shopping-cart/filters/filters.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="side-block">
+<nav class="filter">
+    <div class="row">
```

Comment:
there should ideally be just one nav on a page

Reply:
I have worked on this and will commit the code shortly.